misc/docs/source/tutorial/code_snippets/ray_path_plot.py

```diff
@@ -0,0 +1,7 @@
+from obspy.taup.tau import raypath_plot
+import matplotlib.pyplot as plt
+
+ax = plt.subplot(111, polar=True)
+fig = ax.figure
```

Comment:
just nitpicking.. but matplotlib is nowadays advertising this for most fig/ax setup steps:

```python
fig, ax = plt.subplots(polar=True)
```

See e.g. https://matplotlib.org/examples/pylab_examples/subplots_demo.html

Reply:
Sure! Will change this (also in tau.py)